_database-integrations/mysql/azure/microsoft-azure-mysql-v1.md

```diff
@@ -0,0 +1,173 @@
+  - item: |
+      **MySQL master server** must be in a General Purpose or Memory Optimized pricing tier in Azure.        
+
+# -------------------------- #
+#     Setup Instructions     #
+# -------------------------- #
+
+setup-steps:
+  - title: "Configure database connection settings"
+    anchor: "connect-settings"
+    content: |
+      {% include integrations/templates/configure-connection-settings.html %}
+
+  - title: "Configure Log-based Incremental Replication"
+    anchor: "configure-log-based-incremental-replication"
+    content: |
+      {% include note.html type="single-line" content="**Note**: Skip this step if you're not planning to use Log-based Incremental Replication. [Click to skip ahead](#db-user)." %}
+
+      {% include integrations/databases/setup/binlog/configure-server-settings-intro.html %}
+    substeps:
+      - title: "Configure server settings"
+        anchor: "configure-database-server-settings"
+        content: |
+          In this step, you'll configure your {{ integration.display_name }} server to use Log-based Incremental Replication.
+
+          {% capture server-instructions %}
+          1. In your **Azure portal**, locate your **Azure Database for MySQL** server.
+          2. Click **Settings > Server Parameters**.
+          3. Define the requred server settings using their pre-defined defaults. Refer to the <a href="#server-settings-details" data-toggle="tab">Server settings list</a> tab for explanations of these parameters and their default values. **No other configuration is required on your part.**
+          {% endcapture %}
+          
+          {% include integrations/templates/configure-server-settings.html %}
+          
+      - title: "Retrieve server IDs"
+        anchor: "server-id"
+        content: |
+          {% include integrations/databases/setup/binlog/mysql-server-id.html %}
+
+  - title: "Create a Stitch database user"
+    anchor: "db-user"
+    content: |
+      {% include note.html type="single-line" content="**Note**: You must have the `CREATE USER` and `GRANT OPTION` privileges to complete this step." %} 
+
+      Next, you'll create a dedicated database user for Stitch. This will ensure Stitch is visible in any logs or audits, and allow you to maintain your privilege hierarchy.
+
+      {% include integrations/templates/create-database-user-tabs.html %}
+
+  - title: "Connect Stitch"
+    anchor: "connect-stitch"
+    content: |
+      In this step, you'll complete the setup by entering the database's connection details and defining replication settings in Stitch.
+
+    substeps:
+      - title: "Define the database connection details"
+        anchor: "define-connection-details"
+        content: |
+          {% include shared/database-connection-settings.html type="general" %}
```

Comment:
For the `host` and `port` fields - where would a user find this info? Is it in Microsoft Azure? 

Try looking at the MSSQL doc you started with for a hint.

Reply:
This hasn't been addressed.